core/domain/app_feedback_report_jobs_one_off_test.py

```diff
@@ -0,0 +1,242 @@
+                    self.ANDROID_REPORT_INFO_SCHEMA_VERSION)))
+        current_feedback_report_model.created_on = self.TIMESTAMP_AT_90_DAYS
+        current_feedback_report_model.put()
+
+    def _add_expiring_reports(self):
+        """Adds reports to the model that should be scrubbed."""
+        expiring_android_report_model = (
+            app_feedback_report_models.AppFeedbackReportModel(
+                id=self.ANDROID_REPORT_ID_OVER_90_DAYS,
+                platform=self.PLATFORM_ANDROID,
+                ticket_id='%s.%s.%s' % (
+                    'random_hash', self.TICKET_CREATION_TIMESTAMP.second,
+                    '16CharString1234'),
+                submitted_on=self.REPORT_SUBMITTED_TIMESTAMP,
+                report_type=self.REPORT_TYPE_SUGGESTION,
+                category=self.CATEGORY_OTHER,
+                platform_version=self.PLATFORM_VERSION,
+                device_country_locale_code=(
+                    self.DEVICE_COUNTRY_LOCALE_CODE_INDIA),
+                android_device_model=self.ANDROID_DEVICE_MODEL,
+                android_sdk_version=self.ANDROID_SDK_VERSION,
+                entry_point=self.ENTRY_POINT_NAVIGATION_DRAWER,
+                text_language_code=self.TEXT_LANGUAGE_CODE_ENGLISH,
+                audio_language_code=self.AUDIO_LANGUAGE_CODE_ENGLISH,
+                android_report_info=self.ANDROID_REPORT_INFO,
+                android_report_info_schema_version=(
+                    self.ANDROID_REPORT_INFO_SCHEMA_VERSION)))
+        expiring_android_report_model.created_on = self.TIMESTAMP_OVER_90_DAYS
+        expiring_android_report_model.put()
+
+        expiring_web_report_model = (
+            app_feedback_report_models.AppFeedbackReportModel(
+                id=self.WEB_REPORT_ID_OVER_90_DAYS,
+                platform=self.PLATFORM_WEB,
+                ticket_id='%s.%s.%s' % (
+                    'random_hash', self.TICKET_CREATION_TIMESTAMP.second,
+                    '16CharString1234'),
+                submitted_on=self.REPORT_SUBMITTED_TIMESTAMP,
+                report_type=self.REPORT_TYPE_SUGGESTION,
+                category=self.CATEGORY_OTHER,
+                platform_version=self.PLATFORM_VERSION,
+                device_country_locale_code=(
+                    self.DEVICE_COUNTRY_LOCALE_CODE_INDIA),
+                android_device_model=self.ANDROID_DEVICE_MODEL,
+                android_sdk_version=self.ANDROID_SDK_VERSION,
+                entry_point=self.ENTRY_POINT_NAVIGATION_DRAWER,
+                text_language_code=self.TEXT_LANGUAGE_CODE_ENGLISH,
+                audio_language_code=self.AUDIO_LANGUAGE_CODE_ENGLISH,
+                web_report_info=self.WEB_REPORT_INFO,
+                web_report_info_schema_version=(
+                    self.WEB_REPORT_INFO_SCHEMA_VERSION)))
+        expiring_web_report_model.created_on = self.TIMESTAMP_OVER_90_DAYS
+        expiring_web_report_model.put()
+
+    def test_scrubs_no_models(self):
```

Comment:
Similar to earlier PR, but I suggest naming these tests to specifically describe the expected outcome where possible. Ditto elsewhere.